traffic_editor/package.xml

```diff
@@ -8,6 +8,8 @@
   <license>Apache License 2.0</license>
 
   <build_depend>ament_cmake</build_depend>
+  <build_depend>ament_index_cpp</build_depend>
+  <build_depend>traffic_editor_assets</build_depend>
   <build_depend>libignition-plugin-dev</build_depend>
   <build_depend>libignition-common3</build_depend>
   
```

Comment:
```suggestion

    <exec_depend>traffic_editor_assets</exec_depend>
    
```

`traffic_editor` will crash if the `traffic_editor_assets` package is not there at execution time...

Reply:
Good catch!

Reply:
*** 
https://github.com/osrf/traffic_editor/pull/152/commits/6eed9df290f445eb448ca881046d59816c5ab2de

Caught the exception instead and removed the dependency, printing with a warning